### PR TITLE
Usr/jasteph/reorg sample

### PR DESCRIPTION
**Pull request by jasonstephen15**
Added WPF Sample apps folder, restructured folders -> Added Sample Apps Folder and Getting Started Guides Folder. Moved projects into respective folder.

### CODE REVIEW

File: GettingStartedGuides/WPF_GettingStarted/MainWindow.xaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            <Button x:Name="ButtonGo"
+            DockPanel.Dock="Right"
+            Click="ButtonGo_Click"
+            Content="Go"
+        />
+            <TextBox Name="addressBar"/>
+        </DockPanel>
+        
+        <wv2:WebView2 Name="webView"
+                  Source="https://jasonstephen15.github.io/"
```
Comment (liminzhu) [inline GettingStartedGuides/WPF_GettingStarted/MainWindow.xaml:23]: Would recommend using microsoft.com or bing as opposed to personal site.

Comment (jasonstephen15) [reply GettingStartedGuides/WPF_GettingStarted/MainWindow.xaml:23]: Good catch - def didn't mean that haha, must've been messing around with something - will change it!

File: README.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
 # WebView2 Samples
 
-This repository contains samples that demonstrate the features and usage patterns of [WebView2](https://aka.ms/webview2). As we add more features to WebView2, we will regularly update our samples.
+This repository contains getting started apps as well as sample apps that demonstrate the features and usage patterns of [WebView2](https://aka.ms/webview2). As we add more features to WebView2, we will regularly update our samples.
+
+In the ``GettingStarted`` folder you will find the starter code for its respective guide listed below:
+- [Win32 Getting Started](https://docs.microsoft.com/en-us/microsoft-edge/webview2/gettingstarted/win32)
```
Comment (peiche-jessica) [inline README.md:6]:
```suggestion
- [Win32 Getting Started](https://docs.microsoft.com/microsoft-edge/webview2/gettingstarted/win32)
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
+- [WPF Getting Started](https://docs.microsoft.com/en-us/microsoft-edge/webview2/gettingstarted/wpf)
```
Comment (peiche-jessica) [inline README.md:7]:
```suggestion
- [WPF Getting Started](https://docs.microsoft.com/microsoft-edge/webview2/gettingstarted/wpf)
```

File: README.md [hunk 4]
```diff
@@ -1,8 +1,15 @@
...
+- [WinUI Getting Started](https://docs.microsoft.com/en-us/microsoft-edge/webview2/gettingstarted/winui) (Releasing Soon)
```
Comment (peiche-jessica) [inline README.md:8]: Let's link it when we actually release it :)
```suggestion
- WinUI Getting Started (Releasing Soon)
```

**Pull request merged**